match/lib/match/generator.rb

```diff
@@ -72,7 +72,8 @@ def self.generate_provisioning_profile(params: nil, prov_type: nil, certificate_
         ignore_profiles_with_different_name: true,
         team_id: params[:team_id],
         team_name: params[:team_name],
-        template_name: params[:template_name]
+        template_name: params[:template_name],
+        fail_on_name_taken: true
```

Comment:
Now that I think about it... I’m a little bit iffy on this change since its breaking. I’m going to add a commit into this PR to make this an option on _match_ as well 😊